bootstrap/bootstrap.go

```diff
@@ -75,49 +79,71 @@ func translateInterruptToCancel(ctx context.Context, wg *sync.WaitGroup, cancel
 func Run(
 	ctx context.Context,
 	cancel context.CancelFunc,
-	configDir, profileDir, configFileName string,
-	useRegistry bool,
+	configFileName string,
```

Comment:
I wonder if configFileName should be part of the commonFlags abstraction.

Reply:
It has never been part of commandline flags and always assumed to be `configuration.toml`. I didn't add this. It was on line 78. I just put it on its own line.

The question to you is why did you add it as a parameter rather that use the standard constant of `configuration.toml`

Reply:
Great question.  I guess so it could be varied.  

Since you've created a common command-line args implementation, how would you feel about incorporating a new flag that specifies the configuration file name and defaults to `configuration.toml`.  This would remove it from the parameter list and provide some degree of flexibility during execution to specify alternate configuration file names.

Reply:
I agree with you in general. But this beyond the scope of the PR and what was agreed upon for the refactoring. We should discuss it in Core meeting today and if everyone if ok, then I am ok with add another new common flag.

Reply:
OK, Core WG agrees, so add.